src/test/ui/const-eval/const_panic_libcore.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![no_std]
+#![crate_type = "lib"]
+
+const Z: () = panic!("cheese");
```

Comment:
This appears to require no feature-gate - I don't think this should be insta-stable. Or is it impossible to feature-gate because `panic!` has `allow_internal_unstable`?

Reply:
Curious. I should be able to work around this easily. Maybe const stability of const fns should not be allowed to be circumvented by rustc macros?

Reply:
I tried. I have no idea how this test is still passing. We seem to be having a stability hole.